lib/console.js

```diff
@@ -33,6 +33,10 @@ const metricSummary = function (metrics) {
 Average performance`)
   );
 
+  if (metrics.hasOwnProperty("requestedUrl")) {
```

Comment:
I may be crazy, but is this needed?

Reply:
Because this PR adds `requestedUrl` to the metrics, but not in the same format as the other items with score, etc., the foreach errors out. My goal was to make as few waves in the code as possible, so I chose to remove it.